sdl2-ttf.cabal

```diff
@@ -18,6 +18,11 @@ tested-with:   GHC ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.4 || ==8.10
 
 source-repository head
   type:     git
+
+flag pkgconfig
+  description: Use pkgconfig to find SDL2_ttf
+  manual:      True
+  default:     True
   location: https://github.com/haskell-game/sdl2-ttf
```

Comment:
Thank you for the PR. Could you move the flag one line lower and add an empty line to separate it from the `source-repository` stanza? Please let me know when this is ready to merge.